change-notes/1.23/analysis-csharp.md

```diff
@@ -27,5 +27,6 @@ The following changes in version 1.23 affect C# analysis in all applications.
   picture of the partial flow paths from a given source. The feature is
   disabled by default and can be enabled for individual configurations by
   overriding `int explorationLimit()`.
+* `foreach` statements where the body is guaranteed to be executed at least once, such as `foreach (var x in new string[]{ "a", "b", "c" }) { ... }`, are now recognized by all analyses based on the control flow graph (such as SSA, data flow and taint tracking).
```

Comment:
It might be worth flagging which queries are made more precise, in the "changed queries" section. For example, cs/dereferenced-variable-is-always-null and cs/dereferenced-variable-may-be-null

Reply:
I think that would be too many queries to list, as it would also involve all data flow queries.